<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
SPDX-FileCopyrightText: none
SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-KDE-Accepted-GPL
 </author>
 <class>SubEdit_UI</class>
 <widget class="QWidget" name="SubEdit_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>411</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="buttonPrev">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::LeftArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonNext">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::RightArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonAdd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonCut">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="edit-cut">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonApply">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok-apply">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonStyle">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="format-text-color">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame_position">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="3" column="2" colspan="2">
           <widget class="TimecodeDisplay" name="tc_end" native="true"/>
          </item>
          <item row="4" column="2" colspan="2">
           <widget class="TimecodeDisplay" name="tc_duration" native="true"/>
          </item>
          <item row="0" column="0" colspan="6">
           <widget class="KTextEdit" name="subText">
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="acceptRichText">
             <bool>false</bool>
            </property>
            <property name="checkSpellingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QToolButton" name="buttonLock">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="lock">
              <normaloff>.</normaloff>.</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="2">
           <widget class="TimecodeDisplay" name="tc_position" native="true"/>
          </item>
          <item row="3" column="4" colspan="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_end">
            <property name="text">
             <string>End:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QToolButton" name="buttonOut">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="zone-out">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="buttonIn">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="zone-in">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_position">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Position:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_duration">
            <property name="text">
             <string>Duration:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QToolButton" name="subMenu">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="application-menu">
              <normaloff>.</normaloff>.</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="5">
           <widget class="QLabel" name="char_count">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="5" column="0" colspan="2">
           <widget class="QScrollArea" name="scrollArea">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>284</width>
               <height>348</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="horizontalSpacing">
               <number>0</number>
              </property>
              <item row="12" column="0" colspan="2">
               <widget class="QCheckBox" name="checkOpaque">
                <property name="text">
                 <string>Opaque background</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QCheckBox" name="checkOutlineColor">
                <property name="text">
                 <string>Custom Outline Color</string>
                </property>
               </widget>
              </item>
              <item row="11" column="1">
               <widget class="QComboBox" name="alignment">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QSpinBox" name="fontSize">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="suffix">
                 <string>pt</string>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
                <property name="value">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QCheckBox" name="checkFontSize">
                <property name="text">
                 <string>Custom Font Size</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="KColorButton" name="outlineColor">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="defaultColor">
                 <color>
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="alphaChannelEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <widget class="QSpinBox" name="shadowSize">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="suffix">
                 <string>px</string>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QCheckBox" name="checkPosition">
                <property name="text">
                 <string>Custom Position</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QCheckBox" name="checkFontColor">
                <property name="text">
                 <string>Custom Font Color</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QSpinBox" name="outlineSize">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="suffix">
                 <string>px</string>
                </property>
                <property name="value">
                 <number>4</number>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QCheckBox" name="checkOutlineSize">
                <property name="text">
                 <string>Custom Outline Size</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QCheckBox" name="checkFont">
                <property name="text">
                 <string>Custom Font</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="KColorButton" name="fontColor">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="color">
                 <color>
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
                <property name="defaultColor">
                 <color>
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
                <property name="alphaChannelEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="QFontComboBox" name="fontFamily">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QCheckBox" name="checkShadowSize">
                <property name="text">
                 <string>Custom Shadow</string>
                </property>
               </widget>
              </item>
              <item row="13" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="0">
               <widget class="QCheckBox" name="checkBold">
                <property name="text">
                 <string>Bold</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QCheckBox" name="checkItalic">
                <property name="text">
                 <string>Italic</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="buttonDelete">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="edit-delete">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>TimecodeDisplay</class>
   <extends>QWidget</extends>
   <header>widgets/timecodedisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkFont</sender>
   <signal>toggled(bool)</signal>
   <receiver>fontFamily</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFontSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>fontSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFontColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>fontColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkOutlineColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>outlineColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkOutlineSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>outlineSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkPosition</sender>
   <signal>toggled(bool)</signal>
   <receiver>alignment</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkShadowSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>shadowSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
